packages/cheqd/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@aries-framework/cheqd",
+  "main": "build/index",
+  "types": "build/index",
+  "version": "0.3.3",
+  "files": [
+    "build"
+  ],
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage": "https://github.com/hyperledger/aries-framework-javascript/tree/main/packages/cheqd",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hyperledger/aries-framework-javascript",
+    "directory": "packages/cheqd"
+  },
+  "scripts": {
+    "build": "yarn run clean && yarn run compile",
+    "clean": "rimraf ./build",
+    "compile": "tsc -p tsconfig.build.json",
+    "prepublishOnly": "yarn run build",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@aries-framework/anoncreds": "0.3.3",
+    "@aries-framework/core": "0.3.3",
+    "@cheqd/sdk": "cjs",
+    "@cheqd/ts-proto": "^2.0.1",
```

Comment:
This requires a new release on ts-proto with a similar CommonJS release as SDK above